### PR TITLE
borgbackup: add Python 3.9

### PR DESCRIPTION
**Pull request by balrog-nona**
It builds, installs and publishes fine.
When installed locally, I tested it with
```
borg init --encryption=repokey /path/to/repo
borg create /path/to/repo::Monday ~/src ~/Documents
```
and it worked.

### CODE REVIEW

**Comment by pyhalov** [comment]
Hi. At least borg-35 package should be marked obsolete.

**Comment by balrog-nona** [comment]
> Hi. At least borg-35 package should be marked obsolete.

I created `history` file, thanks.

**Comment by alarcher** [comment]
@pyhalov Should there also be a dependency type=optional to the version deprecating 3.5 to avoid clashes w.r.t /usr/bin/borg

**Comment by pyhalov** [comment]
In fact yes, but it doesn't matter much as the main way of getting updates on OI seems to be just `pkg update`, and it will both fetch  new and remove packages.

File: components/sysutils/borgbackup/history [hunk 1]
```diff
@@ -0,0 +1 @@
+backup/borg-35@1.1.17-2020.0.1.1
```
Comment (AndWac) [inline components/sysutils/borgbackup/history]: This should be "backup/borg-35@1.1.17,5.11-2020.0.1.1"
5.11 is the build version, typically left out when the FMRI is being printed (and separated by a "," from the component version). According to the docs the build version is the version of the OS the component is being built for.

Comment (balrog-nona) [reply components/sysutils/borgbackup/history]: Sorry for that. I took FMRI from `pkg info borg-35` which now I see doesn't contain the whole version.

Comment (AndWac) [reply components/sysutils/borgbackup/history]: Never mind, I made the same mistake in the past and it took some time to understand it.

**Pull request merged**